### DISCUSSION TITLE
Disable tab completion adding trailing backslash

### DISCUSSION THREAD

**Discussion by eggbean** [microsoft/terminal#16478] [Q&A] [answered]
I have been using conhost.exe instead of Windows Terminal just now, as I was editing the `settings.json` for Windows Terminal and I noticed that while all my shell configuration is the same (clink, etc) tab completion was not adding a trailing backslash to paths. I think I prefer it this way and it seems to be something that Windows Terminal is doing (I am surprised about this).

How can I disable Windows Terminal from adding a backslash when tab completing paths?

**Comment by DHowett** [comment, marked as answer]
You know what's even weirder? _We actually don't control tab completion behavior!_

Would you mind sending in a quick side by side comparison video?

**Comment by eggbean** [comment]
Excuse me, but I think I was being delusional (was up all night). There are some occassions where a trailing backslash is appended, but I have a very non-standard setup, with clink, clink functions, a cd replacement called CD Deluxe which nobody uses but is very cool, zoxide, et al. It's nothing to do with the terminal emulator. Sorry about the post 😳

**Reply by DHowett** [reply]
Hey, no problem! Thanks for following up. If you do end up more specifically root-causing the issue, I'd love to hear what it was.
